drivers/sensor/lm73c/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 SER Consulting LLC
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig LM73C
+	bool "LM73C Temperature Sensor"
+	depends on I2C
+	help
+	  Enable I2C-based driver for LM73C Temperature Sensor.
+
+if LM73C
+
+choice
```

Comment:
For new drivers please specify the default power-up configuration through a devicetree property rather than Kconfig.  Kconfig should only select whether the driver is enabled, or provide driver-level non-hardware-related information like whether interrupt or thread support is to be included.

Reply:
OK, was looking at a few other drivers, but actually no idea how to do the dts option, Do you know of an example I can use?

Reply:
It looks like a lot of them have recently been updated; look at 5f605fd5abda4d80209e2d662beef9f82ba29f1a for example.

Reply:
Found one, testing...